changelogs/fragments/javagateway.yml

```diff
@@ -0,0 +1,7 @@
+minor_changes:
```

Comment:
both of this sentences does not bring any value to users of this collection and I would recommend to remove them entirely